File: gm-utils/src/main/java/src/commom/utils/comum/BooleanWrapper.java
package src.commom.utils.comum;

import gm.utils.comum.UBoolean;
import src.commom.utils.object.Null;

public class BooleanWrapper {

	public Boolean value;
	
	public Boolean get() {
		return value;
	}
	
	public BooleanWrapper set(Boolean avalue) {
		this.value = avalue;
		return this;
	}
	
	public boolean isTrue() {
		return UBoolean.isTrue(value);
	}
	
	public boolean isFalse() {
		return UBoolean.isFalse(value);
	}
	
	public boolean isNull() {
		return Null.is(value);
	}
	
}

File: gm-utils/js/src/commom/utils/comum/BooleanWrapper.js
import Null from '../object/Null';

export default class BooleanWrapper {

	get() {
		return this.value;
	}

	set(avalue) {
		this.value = avalue;
		return this;
	}

	isTrue() {
		return UBoolean.isTrue(this.value);
	}

	isFalse() {
		return UBoolean.isFalse(this.value);
	}

	isNull() {
		return Null.is(this.value);
	}

}
